server/src/main/java/org/cloudfoundry/identity/uaa/approval/VmidentityApprovalStore.java

```diff
@@ -13,138 +13,140 @@
 package org.cloudfoundry.identity.uaa.approval;
 
 import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Collection;
```

Comment:
we should make sure that this change also has the switch to actually make this class used. the fact that we somehow lost this, likely is a bug. it was definitely needed to be plumbed (if i remember correctly with default impl. AD user names with long upns didn;t work...)